pip/Cargo.toml

```diff
@@ -12,10 +12,15 @@ license.workspace = true
 [dependencies]
 num-bigint = { workspace = true }
 num-complex = { workspace = true }
-pyo3 = { workspace = true }
 qsc = { path = "../compiler/qsc" }
 miette = { workspace = true, features = ["fancy"] }
 
+[target.'cfg(not(any(target_os = "windows")))'.dependencies]
+pyo3 = { version = "0.19.2", features = ["abi3-py37", "extension-module"] }
```

Comment:
You can still have the dependency in the workspace Cargo.toml and choose the features in the project Cargo.toml (e.g. see `serde` in the ./wasm/Cargo.toml.